Mathlib/Algebra/Group/Equiv/TypeTags.lean

```diff
@@ -113,6 +113,36 @@ and multiplicative endomorphisms of `Multiplicative A`. -/
   { AddMonoidHom.toMultiplicative with
     map_mul' := fun _ _ => rfl }
 
+/-- `Multiplicative (∀ i : ι, K i)` is equivalent to `∀ i : ι, Multiplicative (K i)`. -/
+@[simps]
+def MulEquiv.piMultiplicative (K : ι → Type*) [∀ i, Add (K i)] :
+    Multiplicative (∀ i : ι, K i) ≃* (∀ i : ι, Multiplicative (K i)) where
+  toFun := fun x i ↦ Multiplicative.ofAdd <| Multiplicative.toAdd x i
+  invFun := fun x ↦ Multiplicative.ofAdd fun i ↦ Multiplicative.toAdd (x i)
+  left_inv := fun _ ↦ rfl
+  right_inv := fun _ ↦ rfl
+  map_mul' := fun _ _ ↦ rfl
+
+/-- `Multiplicative (ι → G)` is equivalent to `ι → Multiplicative G`. -/
+abbrev MulEquiv.funMultiplicative (ι) (G) [Add G] :
```

Comment:
```suggestion
variable (ι) (G) in
/-- `Multiplicative (ι → G)` is equivalent to `ι → Multiplicative G`. -/
abbrev MulEquiv.funMultiplicative [Add G] :
```
to make the same variables be reused rather than leaving the universes up to chance